src/DataCollectors/Microsoft.TestPlatform.Extensions.EventLogCollector/Properties/TypesToLoadAttribute.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Microsoft.TestPlatform.Extensions.EventLogCollector;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Utilities;
+
+[assembly: TypesToLoad(typeof(EventLogDataCollector))]
+
+namespace Microsoft.VisualStudio.TestPlatform.ObjectModel.Utilities
+{
+    /// <summary>
+    /// Custom Attribute to specify the exact types which should be loaded from assembly
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Assembly, Inherited = false, AllowMultiple = false)]
+    internal sealed class TypesToLoadAttribute : Attribute
```

Comment:
As mentioned in https://github.com/microsoft/testfx/pull/753 it would be great to change the name of the attribute to `TestExtensionTypesAttribute` and place it in `Microsoft.VisualStudio.TestPlatform` namespace.